include/zim/writer/creator.h

```diff
@@ -205,6 +206,7 @@ namespace zim
         // configuration
         bool m_verbose = false;
         Compression m_compression = Compression::Zstd;
+        int m_compressionLevel = static_cast<int>(ZSTDCompressionLevel::DEFAULT);
```

Comment:
This change the default compression level.
Previously it was `9 | LZMA_PRESET_EXTREME`, now it is `3`. We must keep the same level as before if the user doesn't request anything specific.

Reply:
It's zstd, not LZMA.

Reply:
This is the same problem with zstd.
On LZMA, we were using `19`, now it is `21`. This is a important issue has we already have a (pretty important) bug about this : https://github.com/openzim/libzim/pull/569